addons/mail/static/src/views/web/fields/list_activity/list_activity.xml

```diff
@@ -1,7 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.ListActivity">
-        <ActivityButton record="props.record"/>
-        <span class="o-mail-ListActivity-summary" t-out="summaryText"/>
+        <ActivityButton record="props.record">
+            <span class="o-mail-ListActivity-summary" t-out="summaryText"/>
+        </ActivityButton>
+    </t>
+    <t t-name="mail.ListActivityButton" t-inherit="mail.ActivityButton">
+        <xpath expr="//a[hasclass('o-mail-ActivityButton')]" position="replace">
+            <button class="o-mail-ActivityButton btn btn-link btn-dark p-0 w-100 text-start text-truncate"
+                t-on-click.prevent.stop="onClick" t-ref="button" t-att-aria-label="title" t-att-title="title">
+                <i class="fa fa-fw fa-lg" t-att-class="buttonClass" role="img"/>
```

Comment:
Hello, just a note: it is a bad practice to split the icon prefix (fa) from its main class (fa-clock-o). This makes updating to newer font-awesome very complex, and it also makes interchangeability between fa and oi much harder. In general it's just a maintenance nightmare when both are apart in code.

Reply:
@seb-odoo Hello! Thanks for the note. Split handling for icon prefix and main class was already done prior to this change. Unfortunately it seems that there is a field in `mail.activity.mixin` for the icon string, this case would probably require a refactoring commit + upgrade which is outside of the scope of a simple rectification in the master forward port.
I'd gladly have made the change otherwise.

Cheers!